test_scripts/Smoke/HeartBeat/ATF_HeartBeat_App_does_not_send_HB_and_does_not_respond.lua

```diff
@@ -0,0 +1,97 @@
+--  Requirement summary:
+--  [HeartBeat]: SDL must close only session in case mobile app does not answer on Heartbeat_request
```

Comment:
@istoimenova As I understand for OpenSDL: [[HeartBeat] [GENIVI]: SDL must start HeartBeat process immediately after first StartService request from mobile app](https://adc.luxoft.com/jira/browse/APPLINK-24769?src=confmacro)